src/app/(frontend)/events/[slug]/page.tsx

```diff
@@ -0,0 +1,22 @@
+import { fileURLToPath } from 'url'
+
+export default async function SpecificEventPage({
+  params,
+}: {
+  params: Promise<{ slug: string }>
+}) {
+  const fileURL = `vscode://file/${fileURLToPath(import.meta.url)}`
+  const { slug } = await params
+
+  return (
+    <div className="home">
```

Comment:
minor pedantry here, since these class styles aren't actually defined could we get rid of them as well?
```suggestion
    <div>
```